packages/calcite-components/src/components/input-time-zone/input-time-zone.e2e.ts

```diff
@@ -175,6 +175,60 @@ describe("calcite-input-time-zone", () => {
 
         expect(await timeZoneItem.getProperty("textLabel")).toMatch(testTimeZoneNamesAndOffsets[2].label);
       });
+
+      it("looks up in label and time zone groups (not displayed)", async () => {
+        const searchTerms = ["London", "Belfast", "GMT-12"];
+
+        const page = await newE2EPage();
+        await page.emulateTimezone(testTimeZoneNamesAndOffsets[0].name);
+        await page.setContent(
+          await addTimeZoneNamePolyfill(html` <calcite-input-time-zone></calcite-input-time-zone>`)
```

Comment:
nitpick: remove the extra space after html`

Reply:
Agreed. Need to do a sweep of these. I _think_ my IDE may be adding those extra spaces. 😅